omero/sysadmins/version-requirements.rst

```diff
@@ -111,28 +110,36 @@ Components
      :header-rows: 1
 
      * - PostgreSQL
-       - OMERO 5.3
        - OMERO 5.4
        - OMERO 5.5
+       - OMERO 5.6
      * - 9.3
        - |Deprecated|
-       - |Deprecated|
+       - |Dropped|
        - |Dropped|
      * - 9.4
-       - |Supported|
        - |Supported|
        - |Deprecated|
+       - |Deprecated|
      * - 9.5
        - |Supported|
        - |Supported|
-       - |Supported|
+       - |Deprecated|
      * - 9.6
-       - |Recommended|
-       - |Recommended|
-       - |Recommended|
+       - |Supported|
+       - |Supported|
+       - |Deprecated|
+     * - 10
+       - |Upcoming|
+       - |Supported|
+       - |Supported|
```

Comment:
I'd imagine OMERO 5.6 would recommend at least PG 10.

Reply:
This will require some adjustments to omero-install
PG10 is already supported. We need to review the autogen part. I will review that

Reply:
See https://github.com/ome/omero-install/pull/195

Reply:
Since https://github.com/ome/omero-install/pull/195 makes PG 10 the default should that also be the recommended version for 5.5?